src/sider/senoppfolging/KandidatIkkeSvart.tsx

```diff
@@ -20,7 +23,10 @@ export function KandidatIkkeSvart({ varselAt }: Props) {
       <Heading size="medium">{texts.ikkeSvartHeading}</Heading>
       <BodyShort size="small">{`Den sykmeldte fikk varsel ${tilLesbarDatoMedArUtenManedNavn(
         varselAt
-      )}`}</BodyShort>
+      )} om at det er snart slutt på sykepengene og kan nå svare på spørsmål rundt sin situasjon på innloggede sider.`}</BodyShort>
+      <BodyShort size="small">{`Når spørsmålene er besvart eller hvis den sykmeldte ikke svarer innen ${tilLesbarDatoMedArUtenManedNavn(
+        svarFrist
+      )}, vil du få en oppgave i oversikten om å vurdere videre oppfølging.`}</BodyShort>
```

Comment:
Ikke viktig og kanskje best slik som det er nå. Vil vi legge disse i noe `texts` greier? 🤔

Reply:
Tja, tror jeg foretrekker å bare ha det her i stedet for å måtte ha en texts-funksjon som tar inn dato :man_shrugging: